### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @lucaslattion

### DISCUSSION THREAD

**Discussion by lucaslattion** [heig-vd-dai-course/heig-vd-dai-course#54] [Show and tell]
https://github.com/lucaslattion/java-intellij-idea-and-maven

**Comment by ludelafo** [comment]
Your README file is empty. You could fill it with information regarding the project, how to run and build the app, etc.

I see you kept the `gitignore.bak` files in your repository, and good reflex for the backups. I guess it was just a mistake but they could be deleted as they are not needed by other developers.
